templates/tfengine/components/cicd/terraform.tfvars

```diff
@@ -15,3 +15,5 @@
 billing_account = "{{.billing_account}}"
 project_id      = "{{.project_id}}"
 state_bucket    = "{{.state_bucket}}"
+{{hclField . "service_account" -}}
```

Comment:
The variable name is `service_account_email` but here is `service_account`.

Reply:
Fixed.